src/rest/api.ts

```diff
@@ -78,9 +84,8 @@ export const lagSlettUtkastFetchInfo = (params: FnrFetchParams): FetchInfo => ({
 	method: 'DELETE'
 });
 
-export const lagTaOverUtkastFetchInfo = (params: FnrFetchParams): FetchInfo => ({
-	url: `${VEILARBVEDTAKSSTOTTE_API}/${params.fnr}/utkast/overta`,
-	method: 'POST'
+export const lagTaOverUtkastFetchInfo = (params: { fnr: string; taOverFor: string | undefined | null }): FetchInfo => ({
```

Comment:
Endepunktene for dette ser litt annerledes ut i `veilarbvedtaksstotte`

Reply:
Jeg har gjort endringer i følge med det i backend og har laget pull request på det.